modules/setting/lfs.go

```diff
@@ -22,11 +26,22 @@ var LFS = struct {
 	Storage *Storage
 }{}
 
+// LFSServer represents configuration for hosting Git LFS
+var LFSServer = struct {
+	MaxBatchSize int `ini:MAX_BATCH_SIZE`
```

Comment:
```suggestion
	MaxBatchSize int `ini:"MAX_BATCH_SIZE"`
```

Reply:
I've been running `test-backend`... should probably start running `lint-backend` 😅 

Thanks for your patience as I become more familiar with this project.